src/Analyzers/CSharp/Tests/FixReturnType/FixReturnTypeTests.cs

```diff
@@ -2,353 +2,441 @@
-    }
-}");
+            await VerifyCS.VerifyCodeFixAsync("""
+                class C
+                {
+                    void M()
+                    {
+                        {|CS0127:return|} 1;
+                    }
+                }
+                """, """
+                class C
+                {
+                    int M()
+                    {
+                        return 1;
+                    }
+                }
+                """);
         }
 
         [Fact]
         public async Task Simple_WithTrivia()
         {
-            await TestInRegularAndScript1Async(
-@"class C
-{
-    /*A*/ void /*B*/ M()
-    {
-        [|return|] 1;
-    }
-}",
-@"class C
-{
-    /*A*/
-    int /*B*/ M()
-    {
-        return 1;
-    }
-}");
+            await VerifyCS.VerifyCodeFixAsync("""
+                class C
+                {
+                    /*A*/ void /*B*/ M()
+                    {
+                        {|CS0127:return|} 1;
+                    }
+                }
+                """, """
+                class C
+                {
+                    /*A*/
+                    int /*B*/ M()
+                    {
+                        return 1;
+                    }
+                }
+                """);
             // Note: the formatting change is introduced by Formatter.FormatAsync
         }
 
         [Fact]
         public async Task ReturnString()
         {
-            await TestInRegularAndScript1Async(
-@"class C
-{
-    void M()
-    {
-        [|return|] """";
-    }
-}",
-@"class C
-{
-    string M()
-    {
-        return """";
-    }
-}");
+            await VerifyCS.VerifyCodeFixAsync("""
+                class C
+                {
+                    void M()
+                    {
+                        {|CS0127:return|} "";
+                    }
+                }
+                """, """
+                class C
+                {
+                    string M()
+                    {
+                        return "";
+                    }
+                }
+                """);
         }
 
         [Fact]
         public async Task ReturnNull()
         {
-            await TestInRegularAndScript1Async(
-@"class C
-{
-    void M()
-    {
-        [|return|] null;
-    }
-}",
-@"class C
-{
-    object M()
-    {
-        return null;
-    }
-}");
+            await VerifyCS.VerifyCodeFixAsync("""
+                class C
+                {
+                    void M()
+                    {
+                        {|CS0127:return|} null;
+                    }
+                }
+                """, """
+                class C
+                {
+                    object M()
+                    {
+                        return null;
+                    }
+                }
+                """);
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/33481")]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/65302")]
```

Comment:
unskip.  we dont' skip tests if they don't have hte outcome we want.  we still have the test, just demonstrating the current behavior.  a comment in teh code can mention if this is undesirable behavior and that it's ok if it changes.

Reply:
If I unskip it and return fixed code back to "fix" the return type to `object` then `VerifyCS` would fail the test because it contains compilation errors after the fix